### ISSUE TITLE
Custom host WSS connections fail to connect on NodeJS

### ISSUE THREAD

**Issue by Akhawais** [pusher/pusher-js#494]
**Do you want to request a *feature* or report a *bug*?**
Bug

**What is the current behavior?**
Connecting to a laravel-websockets server in a node.js application fails and state changes to `unavailable`. Server doesn't show any indication of a client connecting. Same config, wss url, etc. works fine in the browser runtime. After some digging, I first thought the fault lies somewhere with `faye-websocket` (only referenced for the Node runtime) and TLS connection errors.

However, weirdly, I am able to fix the problem by replacing [this line](https://github.com/pusher/pusher-js/blob/9152c97631d08d428d36ded807b8c81ef7c008d3/dist/node/pusher.js#L10095) with the following:
```js
var websocket = require('faye-websocket');
```
i.e. bypassing webpack's own require function.

Any ideas as to why the webpack compiled version of faye-websocket is having trouble connecting? The webpack/config.node.js looks fine.

Using version 7.0.0, also tried with 6.0.3. On macOS and Ubuntu as well. Tried with nodejs 14 and 12 as well.

```js
const ps = new Pusher(config.pusherKey, {
  wsHost: config.wsHost,
  enabledTransports: ['ws', 'wss'],
  disableStats: true,
  forceTLS: true,
});
```
This is the config I'm using however I don't think it would affect anything, as it's not passed to the WebSocket client constructor. 

Many thanks!

**Comment by leesio** [comment]
> However, weirdly, I am able to fix the problem by replacing this line with the following...

That's odd. Do you know what version of `faye-websocket` is used when you add that `require`? Maybe this could be solved by upgrading the version of `faye-websocket` we bundle with the library.

**Comment by leesio** [comment]
I'm still struggling to reproduce this issue so I'm going to close it. If you can provide any further information, drop a message here and I'll reopen it.

**Comment by Akhawais** [comment]
Apologies I missed your first reply. The `require`d version of `faye-websocket` is 0.11.3 and the one bundled with the library is 0.9.4. The machine I'm deploying to is Windows 10 running node 12.18.3 and npm 6.14.6.

I've just added 0.9.4 specifically and the issue still persisted, so the solution here would be to upgrade the version used - I'll submit a PR shortly. Thanks!

**Comment by Akhawais** [comment]
Hi @leesio, I hope you're doing well.

Thank you for accepting the earlier PR. I've just finished up the app I'm using this for, and when deploying I noticed that the problem has come back! It appears the `dist/node/pusher.js` file hasn't been re-built and just the version constant has changed (looking at the last commit https://github.com/pusher/pusher-js/commit/22e886e3ba2bb7fdff0a9a7aeb9534bfb13cffa2#diff-b8373734d6a61ce9f58a34a2ee45dbd6fe88c266ef82e961361ffa69694ccef3)

I think perhaps the CI failed to run successfully?
